### PR TITLE
Add unequal split

### LINKED ISSUE #3688: Expenditures: As a user I can use split funds unequal
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to use unequal split option. User can set a max amount of funds, change currency, add/delete and select recipients. User can display a reserve bar, set a percentage use of max funds for each of recipient by using a slider bar and see a share below.

## Acceptance Criteria

- an option to set max amount of funds
- an option to change currency
- an option to display a reserve bar
- an option to add/delete recipients
- an option to select a recipient
- an option to display and use slider bar under each recipient
- an option to display a share

## Design

- [Figma Link - Unequal Distribution no recipients](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A285480)
- [Figma Link - Unequal Distribution with recipients](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3802%3A278705)

Screenshot 2022-07-27 120047

Screenshot 2022-07-27 115817

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds unequal split option.
When split is set to "unequal", followed form fields are displayed: 
- input to set max amount of funds
- select to change currency
- reserve bar
- recipients array: 
    - select to choose a recipient
    - slider bar to set a percentage use of max funds for the recipient 
    - share of max funds for the recipient

Screenshot 2022-08-04 at 11 40 01

[Figma Link - Unequal Distribution no recipients](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A285480)
[Figma Link - Unequal Distribution with recipients](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3802%3A278705)

Resolves #3688

### CODE REVIEW

**Review by willm30 — APPROVED** [review]
Working well for me.

**Review by Julianahlk — APPROVED** [review]
Works as expected! 👍
Screen Shot 2022-08-05 at 12 52 59 AM
One thing regarding the Amount input, if you toggle to Equal and toggle back to Unequal, the state of everything else is saved except for the amount.

File: src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx [hunk 1]
```diff
@@ -26,118 +39,211 @@ const MSG = defineMessages({
...
+                // mock, needs to be changed to the actual value
+                maxAmount: '0',
+                fieldName: 'split.amount.value',
+              }}
+            />
+          </div>
+          <div className={styles.tokenWrapper}>
+            <div>
+              <TokenSymbolSelector
+                label=""
```
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx:126]: Please always include a label, the label is used for accessibility purposes

File: src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx [hunk 2]
```diff
@@ -26,118 +39,211 @@ const MSG = defineMessages({
...
+              return (
+                <FormSection
+                  appearance={{ border: 'bottom' }}
+                  key={recipient?.key}
+                >
+                  <div className={styles.recipientWrapper}>
+                    <div>
+                      <UserPickerWithSearch
+                        data={colonyMembers?.subscribedUsers || []}
+                        label=""
```
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx:162]: Same here, add `elementOnly` to this component and then add a label

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Looking good! Just some small comments regarding the labels

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me! Good job

**Pull request merged**